src/main/groovy/io/cify/framework/core/Capabilities.groovy

```diff
@@ -32,18 +35,34 @@ class Capabilities {
      * @return Capabilities
      */
     static Capabilities parseFromJsonString(String capabilitiesJson) {
-        LOG.debug(MARKER, "Parse capabilities from json string \n $capabilitiesJson")
-        return new JsonSlurper().parseText(capabilitiesJson) as Capabilities
+
+        def json = new JsonSlurper().parseText(capabilitiesJson)
+        List<Map> parsedList = []
+        try {
```

Comment:
Short comment what is happening here would be usefull

Reply:
Done